templates/email/report.html

```diff
@@ -0,0 +1,367 @@
+
```

Comment:
Since this HTML file is derived from `report.mjml`, let's not check it into the repo.

Reply:
I adjusted this branch to be on top of #55 so the diff is sane to look at for now

We included the rendered mjml file in case we needed to get the emails sent before a more complex integration with MJML can get stitched up. What's the timing looking like on that?

We could add `mjml` to the `package.json` in the root of the repo, and then as long as `npm install` is run before the email script, the email script could invoke `mjml /path/to/repo/reports/template.mjml -s` to get the rendered MJML output to STDOUT for easy capture and then do some placeholder replacement